llvm/docs/LangRef.rst

```diff
@@ -3076,6 +3084,7 @@ specifications are given in this list:
 -  ``v64:64:64`` - 64-bit vector is 64-bit aligned
 -  ``v128:128:128`` - 128-bit vector is 128-bit aligned
 -  ``a:0:64`` - aggregates are 64-bit aligned
+-  ``z:0`` - default null value of 0 for unlisted(INT_MAX) address space
```

Comment:
What would the "unlisted" AS be?

Reply:
Unlisted here means, for address spaces which are not specified in the data layout string.